### PR TITLE
Aleks and Nathan boilerplate changes

### PR DESCRIPTION
**Pull request by nkuik**

### CODE REVIEW

File: week4/homework-template/.gitignore [hunk 1]
```diff
@@ -1,46 +1,46 @@
...
-.vscode/settings.json
-
-# React build
-build
-
-dist/
-
-.DS_Store
-
+# Logs
```
Comment (nkuik) [inline week4/homework-template/.gitignore:1]: It apparently thinks these files below (which are all almost exactly the same) are completely new. Has something to do with unix line endings, I suspect.

File: week4/homework-template/package.json [hunk 2]
```diff
@@ -1,34 +1,35 @@
...
+    "body-parser": "^1.18.3",
+    "cookie-parser": "~1.4.3",
+    "cors": "^2.8.4",
+    "debug": "~2.6.0",
+    "disqus-react": "^1.0.5",
+    "dotenv": "^4.0.0",
+    "express": "^4.16.4",
+    "express-brute": "^1.0.1",
+    "express-form-data": "^2.0.1",
+    "knex": "^0.21.1",
```
Comment (benna100) [inline week4/homework-template/package.json:26]: They are not entirely the same, this line is fx added

File: week4/homework-template/sql_baseline.sql [hunk 3]
```diff
@@ -0,0 +1,46 @@
+CREATE DATABASE IF NOT EXISTS `mealsharing`;
```
Comment (benna100) [inline week4/homework-template/sql_baseline.sql:1]: This is a really good idea. Hopefully they already have this, but if not

File: week4/homework-template/src/backend/api/meal.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
+const express = require("express");
+const app = express();
```
Comment (benna100) [inline week4/homework-template/src/backend/api/meal.js:2]: We dont use this `app`

**Review by benna100 — COMMENTED** [review]
Looks good :) Small changes

**Pull request closed without merging**